app/javascript/src/components/editor/Modals/TranslationKeysModal.svelte

```diff
@@ -28,6 +32,109 @@
     selectedKey = value
     newKeyInput.value = ""
   }
+
+  const DELIMITER_TO_FILE_TYPE = new Map([
+    [",", "CSV"],
+    ["\t", "TSV"],
+    [null, "TSV/CSV"]
+  ])
+
+  async function importKeysFromClipboard() {
```

Comment:
Since this is quite a destructive function, it might be good to remind the user that they are about to override their current keys.

```js
if (!confirm("Importing will override your current translations keys")) return
```

Reply:
I was thinking since the keys aren't fully overwritten until the user saves, then there is always a way back.

Also, I don't really like the idea of the browser dialog leading directly to copying to clipboard (especially since the button is just labeled "Confirm").
Maybe if it were its own special in-app modal with a button like "I get it, confirm anyways"?